CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 - `[tendermint-abci,tendermint-rpc]` Fix DeliverTx response deserialization
   issues with `gas_wanted` and `gas_used` fields ([#876])
+- `[tendermint]` Update TendermintConfig for Tendermint v.0.34.x ([#896])
```

Comment:
Please add this entry as a separate file in the `.changelog/unreleased/bug-fixes` folder.

See <https://github.com/informalsystems/tendermint-rs/blob/master/CONTRIBUTING.md#changelog> for details.

Reply:
Thank you too,`unclog` looks very handy for this! 

Just a note, the format shown in CONTRIBUTING shows `- [pkg] \#xxx Some description about the change`, but the other entries in `.changelog` had a slightly different format, so I followed that for consistency.

Reply:
Good point, I'll update that in the contributing guidelines :+1: